### PR TITLE
Close journal fix

### PR DESCRIPTION
**Pull request by dragotin**
More fixes to handle the case that a database connect was lost because of hibernation and removeable disks.

@ogoffart pls review.

### CODE REVIEW

File: src/libsync/propagateremotemove.cpp [hunk 1]
```diff
@@ -154,15 +154,21 @@ void PropagateRemoteMove::finalize()
 {
     SyncJournalFileRecord oldRecord =
             _propagator->_journal->getFileRecord(_item->_originalFile);
+    // if reading from db failed still continue hoping that deleteFileRecord
+    // reopens the db successfully.
+    // only the content checksum stuff is taken from the old record, which
+    // is ok to loose it.
```
Comment (ogoffart) [inline src/libsync/propagateremotemove.cpp]: grammar

**Comment by ogoffart** [comment]
:+1: :

**Comment by dragotin** [comment]
Comment fixed and force pushed.

**Comment by ogoffart** [comment]
:+1:

**Pull request merged**